xep-0383.xml

```diff
@@ -241,7 +241,39 @@
   </section2>
 </section1>
 <section1 topic='XML Schema' anchor='schema'>
-  <p>TODO.</p>
+  <code><![CDATA[
+<?xml version='1.0' encoding='UTF-8'?>
+
+<xs:schema
+    xmlns:xs='http://www.w3.org/2001/XMLSchema'
+    targetNamespace='urn:xmpp:burner:0'
+    xmlns='urn:xmpp:burner:0'
+    elementFormDefault='qualified'>
+
+  <xs:annotation>
+    <xs:documentation>
+      The protocol documented by this schema is defined in
+      XEP-0383: http://www.xmpp.org/extensions/xep-0383.html
+    </xs:documentation>
+  </xs:annotation>
+
+  <xs:element name='identity'>
+    <xs:complexType>
+      <xs:sequence minOccurs="0" maxOccurs="1">
+        <xs:element name='jid' type='bareJIDType'/>
+      </xs:sequence>
+    </xs:complexType>
+  </xs:element>
+
+  <xs:simpleType name='bareJIDType'>
+    <xs:restriction base='xs:string'>
+      <xs:minLength value='5'/>
+      <xs:maxLength value='2047'/>
```

Comment:
Oh good catch; I'm not really sure what to do about this one. I suspect many of our schemas have this issue (since I think I copy/pasted this from somewhere else and tweaked the numbers).